java/src/main/java/bindings/kvikio/cufile/CuFileDriver.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ package bindings.kvikio.cufile;
```

Comment:
Is there code formatting that can be applied? There are some extraneous spaces in this PR.
```suggestion
package bindings.kvikio.cufile;
```

Reply:
Thanks, these should be resolved now.